src/battle-dex.ts

```diff
@@ -711,7 +721,7 @@ const Dex = new class implements ModdedDex {
 			spriteData.w *= 2;
 			spriteData.h *= 2;
 			spriteData.y += -22;
-		} else if ((species.isTotem || isDynamax) && !options.noScale) {
+		} else if (species.isTotem && !options.noScale) {
```

Comment:
```suggestion
		} else if ((species.isTotem || isDynamax) && !options.noScale) {
```

It's possible to turn AFD mode off, and your code needs to account for that.